test/e2e/ratelimit/ratelimit_universal.go

```diff
@@ -160,34 +181,30 @@ destinations:
 conf:
   http:
     requests: 1
-    interval: 10s
-`
+    interval: %[1]s
```

Comment:
Why is the format different here?

Reply:
it's the format for reusing one of the `Sprintf` arguments